Mathlib/RingTheory/Kaehler/JacobiZariski.lean

```diff
@@ -31,11 +31,12 @@ open KaehlerDifferential TensorProduct MvPolynomial
 
 namespace Algebra
 
-universe u₁ u₂ u₃ u₄ w' w u v uT
+universe d e f g h i a b c
```

Comment:
Can you please add a comment that this order is deliberate? Otherwise, somebody might want to come along and "clean this up".